tests/Functional/ORMModelFactoryTest.php

```diff
@@ -540,6 +545,102 @@ public function can_count_based_on_a_relationship(): void
         self::assertSame(2, PostFactory::count(['category' => $category]));
     }
 
+    /**
+     * @test
+     */
+    public function can_create_factories_for_rich_domain_models(): void
```

Comment:
This test passes and adds the comments via `RichPost::setComment()` (property-accessor).